example/ios/Podfile.lock

```diff
@@ -58,25 +58,25 @@ DEPENDENCIES:
   - React/RCTText (from `../../node_modules/react-native`)
   - React/RCTVibration (from `../../node_modules/react-native`)
   - React/RCTWebSocket (from `../../node_modules/react-native`)
-  - Yoga (from `../../node_modules/react-native/ReactCommon/yoga/Yoga.podspec`)
+  - Yoga (from `../../node_modules/react-native/ReactCommon/yoga/yoga.podspec`)
 
 EXTERNAL SOURCES:
   React:
-    :path: ../../node_modules/react-native
+    :path: "../../node_modules/react-native"
   react-native-google-maps:
-    :path: ../../
+    :path: "../../"
   react-native-maps:
-    :path: ../../
+    :path: "../../"
   Yoga:
-    :path: ../../node_modules/react-native/ReactCommon/yoga/Yoga.podspec
+    :path: "../../node_modules/react-native/ReactCommon/yoga/yoga.podspec"
 
 SPEC CHECKSUMS:
-  GoogleMaps: a5b5bbe47734e2443bde781a6aa64e69fdb6d785
+  GoogleMaps: c087b8e5dfe87ca6ebf59adb9b4894a4146bec4f
   React: 0c9191a8b0c843d7004f950ac6b5f6cba9d125c7
-  react-native-google-maps: 73a7f74a00d36c74df41388ebb8394ceba81ef0e
-  react-native-maps: ee0bb36520eb49dfc2bf11754a486a4fa5c5f883
+  react-native-google-maps: 57af8d96a008489af2657caaeb074e7448451ff8
+  react-native-maps: 016cff46e0ae7ad4c1b1143a1aa2044e9983a544
   Yoga: 89c8738d42a0b46a113acb4e574336d61cba2985
 
-PODFILE CHECKSUM: c069397afd4c9ecf6886b7c00e7d7264dced43a4
+PODFILE CHECKSUM: d8c939699ce1f7d87205849a333b7e27fb1c4725
 
-COCOAPODS: 1.3.1
+COCOAPODS: 1.2.0
```

Comment:
Is there a reason we're downgrading cocoapods?

Reply:
I think that's what I have on my system, I can try to update cocoapods